src/main/java/com/sarahhand/fractals/view/FractalsUI.java

```diff
@@ -41,23 +47,38 @@
 	private final int LEFT_CLICK = 1;
 	private final int RIGHT_CLICK = 3;
 
+	/** This method sets up the UI for the 
+	 */
 	public FractalsUI() {
 		frame = new JFrame("Fractal Viewer");
-		
 		frame.setLayout(new FlowLayout());
 		frameDimension = new Dimension(FRAME_WIDTH, FRAME_HEIGHT);
-		viewerFactory = new MandelbrotViewerFactory();
 		frame.setSize(frameDimension);
+		frame.setResizable(false);
 		frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+
+		viewerFactory = new MandelbrotViewerFactory();
 		viewer = viewerFactory.createViewer();
 		mandelConfig = viewer.getConfig();
 		image = new ImageIcon(viewer.getView(frameDimension));
 		imageLabel = new JLabel(image);
+
+		saveImage = new JButton("Save");
+		loadImage = new JButton("Load");
+
+		saveImage.addActionListener(this);
```

Comment:
I switched them to be inner classes because they are too strongly coupled to the FractalsUI to be in separate classes.